client/lib/screen-title/index.js

```diff
@@ -0,0 +1,28 @@
+/**
+ * Internal dependencies
+ */
+import { setTitle as legacySetTitle } from './actions';
+
+/**
+ * Given a Redux store instance, subscribes to the store, updating the page
+ * when the title state changes. Currently, this dispatches through the legacy
+ * Flux actions used elsewhere in the codebase.
+ *
+ * @param {Object} store Redux store instance
+ */
+export function subscribeToStore( store ) {
```

Comment:
Two alternatives to consider here. Perhaps this is still the correct approach, but curious your thoughts:

- Create a Redux middleware which, when an action type for setting title or unread count, updates the document title automatically.
- Override the store dispatch method to perform a similar check as the first point, in which a title or unread count set action changes the document title.
 - Doesn't require title middleware to be specified in Redux initialiation module
 - A bit uglier to override dispatch, but the Redux middleware implementation is very similar to this
 - Example: https://github.com/aduth/regimen/blob/e5d5749db0628b8e83e56c78891255506113cd7c/src/lib/standalone/index.js#L51-L65